pixi.toml

```
@@ -53,15 +53,18 @@ lint = { depends_on = [
     "mypy-ribasim-api",
 ] }
 # Build
-build-ribasim-cli = { cmd = "cd build/create_binaries && julia --project create_app.jl", depends_on = [
+build-ribasim-cli = { cmd = "cd build/create_binaries && julia --project build.jl --app", depends_on = [
     "generate-testmodels",
     "instantiate-julia",
 ] }
-build-libribasim = { cmd = "cd build/create_binaries && julia --project create_lib.jl", depends_on = [
+build-libribasim = { cmd = "cd build/create_binaries && julia --project build.jl --lib", depends_on = [
+    "generate-testmodels",
+    "instantiate-julia",
+] }
+build = { cmd = "cd build/create_binaries && julia --project build.jl --app --lib", depends_on = [
```

```suggestion
build = { "cmd" = "julia --project build.jl --app --lib", cwd = "build/create_binaries", depends_on = [
```

Now that we can use the newest pixi we can clean this up a bit. Probably good to run a formatter on it afterwards